packages/serial/src/serialapi/transport/SendDataBridgeMessages.ts

```diff
@@ -310,9 +310,13 @@ export class SendDataBridgeRequestTransmitReport
 				"transmit status":
 					getEnumMemberName(TransmitStatus, this.transmitStatus)
 					+ (this.txReport
+							&& (this.transmitStatus === TransmitStatus.OK
+								|| this.transmitStatus === TransmitStatus.NoAck)
```

Comment:
I think we should keep the txTicks in the log message always. There are some edge cases with older controllers which respond with a `Fail` status after a while - probable due to errors in route resolution or so. This is functionally different from the case where the controller immediately responds with `Fail`.